examples/epstein_civil_violence/epstein_civil_violence/model.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, height=40, width=40, citizen_density=0.7, cop_density=0.074,
                                   threshold=self.active_threshold,
                                   vision=self.citizen_vision)
                 unique_id += 1
-                self.grid[y][x] = citizen
+                self.grid[x, y] = citizen
```

Comment:
Shouldn't this be `y, x`?

Reply:
Yes and no: It should be `y, x` to not introduce a change, but I think the current `[y][x]` is a bug: Earlier (line 83) `(x, y)` is explicitly passed as a `pos` parameter to citizen

Reply:
In that case, the example needs to be fixed, separate from this PR. This PR will take a long time to be merged.

Reply:
Theoretically I would agree, but since this is just an example code and there is no behavioral change to the model itself I would say this change should be okay. If you and others insist on keeping it with `(y, x)` I don't mind changing it back at all, but honestly I currently don't have the time nor the desire to create a separate PR and rebase this one just to have a "clean" solution.
I agree that this PR will take a long time to be merged (if ever), but I certainly don't think *this* is the cause

Reply:
Back in the day, I proposed to change `x, y` with `col, row` in order to prevent certain confusion in the grid addressing. What made sense 4 or 5 years ago could still be implemented, with great benefit.

Reply:
Pinging @jackiekazil, @tpike3, @wang-boyu  for before the 1.0 release. Just wondering if we should switch to `self.grid[x][y] = agent` instead of `self.grid[y][x] = agent` in 1.0 ASAP. The convention affects the examples only.

Reply:
> Back in the day, I proposed to change `x, y` with `col, row` in order to prevent certain confusion in the grid addressing. What made sense 4 or 5 years ago could still be implemented, with great benefit.

I would prefer this - use variables `(x, y)` for coordinates, and `[row, col]` for matrix/grid indexing (so we don't need to change too much code hopefully). We can highlight the relations between these variables in documentations.

If needed, we could perhaps also provide utility functions to do such conversion, e.g., `xy_2_rowcol()`, or with some better function name.

Reply:
It's a quick fix. I have made #1366.

Reply:
For reference, Agents.jl defines the first index as the first dimension, second index as the second dimension, ... up to N. This is more general, and we shouldn't be bogged down to a 2D-specific representation detail such as col/row. Think [Einstein notation](https://en.wikipedia.org/wiki/Einstein_notation) for tensor.

Reply:
Oh sorry I think I misunderstood your question. My previous comment applies only to the upcoming `RasterLayer` in Mesa-Geo, not Mesa's grid space, since we're not viewing the grid space as matrix here.

It would definitely make sense to use `self.grid[x][y]` in all examples to make it consistent.